media_driver/linux/common/codec/ddi/media_ddi_decode_vp8.cpp

```diff
@@ -84,12 +84,17 @@ VAStatus DdiDecodeVP8::ParseSliceParams(
     picParams->CodedCoeffTokenPartition = (num_token_partitions != 8) ? (num_token_partitions >> 1) : 3;
     //macroblock_offset is in unit of bit.it should be always the next byte, the byte is divided to two parts
     //used bits and remaining bits, if used bits == 8, uiFirstMbByteOffset should add 1, so use 8 to do the ceil operator
```

Comment:
what about the comment above, is it still actual ? especially, I'm confused about this "if used bits == 8, uiFirstMbByteOffset should add 1, so use 8 to do the ceil operator"